doc/src/manual/variables.md

```diff
@@ -111,16 +111,35 @@ variable name. For example, if `+ᵃ` is an operator, then `+ᵃx` must be writt
 it from `+ ᵃx` where `ᵃx` is the variable name.
 
 
-A particular class of variable names is one that contains only underscores. These identifiers can only be assigned values but cannot be used to assign values to other variables.
-More technically, they can only be used as an [L-value](https://en.wikipedia.org/wiki/Value_(computer_science)#lrvalue), but not as an
- [R-value](https://en.wikipedia.org/wiki/R-value):
+
+A particular class of variable names is one that contains only underscores. These identifiers can only be assigned values but cannot be used to assign values to other variables (or examined on their own). More technically, they can only be used as an [L-value](https://en.wikipedia.org/wiki/Value_(computer_science)#lrvalue), but not as an [R-value](https://en.wikipedia.org/wiki/R-value) or [non-L-value](https://en.wikipedia.org/wiki/Value_(computer_science)) (**non-L-value** is colloquially known as **R-value**):
 
 ```julia-repl
-julia> x, ___ = size([2 2; 1 1])
+julia> x, ___ = size([2 2; 1 1])                         # ___ is used as an L-value
 (2, 2)
 
-julia> y = ___
+julia> y = ___                                           # ___ is used as an R-value
 ERROR: syntax: all-underscore identifier used as rvalue
+
+julia> ___                                               # ___ is used as a non-L-value
+ERROR: all-underscore identifier used as rvalue
+```
+
+In Julia, a particular class of variable names is used when you only want a specific part of a collection (which has many values), and wants to "throw away" the rest of the values in the collection (regardless of the number)." Unlike languages like Python, the particular class of variables can not be examined on their own:
+```julia-repl
+julia> student1 = "Comprehensive High School", "Peter Pan", 13, "Grade 12", "Science";
```

Comment:
I think what "a particular class of variable names" refers to has already been established. It is probably better to refer to it as "underscore variables" or something along those lines after that. 

Also, this to me, makes it sounds like it is something special about `_` in unpacking syntax but there is no real difference in

```
julia> a,b,_ = 1,2,3,4
(1, 2, 3, 4)

julia> a,b,c = 1,2,3,4
(1, 2, 3, 4)
```

except the properties of `_` that has already been stated.

This particular example of using `_  in unpacking is already shown above when `x, ___ = size([2 2; 1 1])` where the tuple (2,2)  is unpacked into `x` and `___`.

In summary, I think this can be significantly shortened.

Reply:
So what are you suggesting? That the PR is closed so we move in favour of [#45964](https://github.com/JuliaLang/julia/pull/45964) or that the file is changed and shortened to be better understood?